doc/INSTALL_ru-RU.md

```diff
@@ -0,0 +1,101 @@
+# Выберите свой способ установки
+
+1. [Установка из бинарных файлов](#Install-from-binaries)
+
+   Это самый простой способ, который рекомендуется для большинства пользователей.
+
+2. [Сборка из исходного кода](#Build-from-source)
+
+   Таким образом, вы можете получить самые последние неизданные функции и подготовиться к участию. Это требует дополнительных шагов, и это единственный выбор, если ваша ОС официально не поддерживается.
+
+3. [Использовать Артефакты](#Использовать-Артефакты)
+
+    Таким образом, вы можете получить самые последние неизданные функции, не создавая их самостоятельно. Но не все операционные системы поддерживают это, и это доступно только для обновления, а не для первой установки.
+
+4. [Build from AUR](#Build-from-AUR)
```

Comment:
Is this hard to translate? :thinking: I think it should be similar to "Build from source".

Reply:
I haven't found any reliable Russian-language sources for ArchLinux users, so let's keep It that way for now.